.misc/travis_build_and_test.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+if [${UNSUPPORTED} == "false"] || [${TRAVIS_BRANCH} == "master"]; then
```

Comment:
I'm pretty sure you need spaces after and before the `]` and `[`

Reply:
see https://travis-ci.org/coala/coala/jobs/301516951#L688

Reply:
also consider using `[[`. it's generally safer for bash

Reply:
Hey, @Mixih why does by build fail when I change `[` to `[[`?